### PR TITLE
Add splash screen

### PR DESCRIPTION
**Pull request by klikooo**

### CODE REVIEW

File: app/src/main/res/layout/activity_splash_screen.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    android:background="@color/govDarkBlue">
+
+    <ImageView android:id="@+id/splashscreen" android:layout_width="wrap_content"
+        android:layout_height="fill_parent"
+        android:src="@drawable/voting_pass_icon"
+        android:layout_gravity="center"/>
+
+    <TextView android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:text="Hello World, splash"/>
```
Comment (wkmeijer) [inline app/src/main/res/layout/activity_splash_screen.xml]: This textview is not visible and can be removed.

File: app/src/main/java/com/digitalvotingpass/digitalvotingpass/SplashActivity.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    /**
+     * Create splash screen/
+     * @param bundle
+     */
+    @Override
+    public void onCreate(Bundle bundle) {
+        super.onCreate(bundle);
+        setContentView(R.layout.activity_splash_screen);
+
+        //Start MainActivity after SPLASH_DISPLAY_LENGTH
```
Comment (wkmeijer) [inline app/src/main/java/com/digitalvotingpass/digitalvotingpass/SplashActivity.java:25]: Might be best to add a comment here that the delay can be removed when there is actual loading to be done.

**Review by wkmeijer — CHANGES_REQUESTED** [review]
Works great, please add a textview that displays "Digital voting pass" as seen in the prototype disign: https://marvelapp.com/39b062h/screen/28342702

Further: see comments

File: app/src/main/res/layout/activity_splash_screen.xml (hunk 1 above)
Comment (klikooo) [reply app/src/main/res/layout/activity_splash_screen.xml]: Hah missed that one

**Comment by klikooo** [comment]
Fixed comments

**Review by wkmeijer — APPROVED** [review]

**Pull request merged**